workloads/hyper-scale/README.md

```diff
@@ -0,0 +1,31 @@
+# Hypershift perfscale Scripts
+
+The purpose of the script is to install Hypershift Operator on an Openshift cluster and create HostedCluster workload on them.
+
+1. Install and create cluster: `./run.sh build`
+2. Cleanup clusters: `./run.sh clean`
+
+
+Running from CLI:
+
+```sh
+$./run.sh build
+```
+
+## Workload variables
+
+The run.sh script can be tweaked with the following environment variables
+
+| Variable                | Description              | Default |
+|-------------------------|--------------------------|---------|
+| **AWS_REGION**            | AWS region where the management cluster lives on | us-west-2 |
+| **AWS_ACCESS_KEY_ID**       | AWS access key for authentication | **REQUIRED** |
+| **AWS_SECRET_ACCESS_KEY**     | AWS secret for authentication | **REQUIRED** |
+| **ROSA_ENVIRONMENT**           | ROSA environment, staging or production        | staging |
```

Comment:
At the moment it doesn't look like we actually need any of the rosa bits in here since we are not actually using the binary for anything productive.

Reply:
Right. This wasn't a comment to the AWS account info but the actual rosa bits which aren't used at this point. We can leave them in there as a place holder but we should provide a comment around them saying such